Example/Tests/Matchers/CDTQContainsAllElementsMatcher.h

```diff
@@ -0,0 +1,18 @@
+//
+//  CDTQContainsAllElementsMatcher.h
+//  CloudantQueryObjc
+//
+//  Created by Rhys Short on 21/01/2015.
+//  Copyright (c) 2015 Cloudant Inc. All rights reserved.
+//
+
+#ifndef CloudantQueryObjc_CDTQContainsAllElementsMatcher_h
+#define CloudantQueryObjc_CDTQContainsAllElementsMatcher_h
+
+#import "Expecta.h"
+
+EXPMatcherInterface(containsAllElements, (NSArray * expected));
+
+#define containAllElements containsAllElements
```

Comment:
I think so, I was following the pattern in the expecta [readme] (https://github.com/specta/expecta/blob/master/README.md#writing-new-matchers), although the names could probably be improved